blog/articles/article_detailed/article_visual_debugger_in_vscode/debugger.md

```diff
@@ -85,20 +106,24 @@ env vars are inputtable for different oses in settings.json
     }
 }
 ```
-```
+
+```shell
 $ python3 file.py 
 Example(foo=10, bar='abc', is_smth=False)
 ```
 
-Those env vars will be available for access from within the launched web server and unit tests during visual debug usage too
+Those env vars will be available for access from within the launched web server and unit tests during visual debug usage too.
 
 # Python with running debug for django
 
+```shell
 cd ../django_example && code -r .
+```
 
-set a breakpoint in a view at urls.py and launch Python Django debug, like in this picture below
+set a breakpoint in a view at `urls.py` and launch Python Django debug, like in this picture below
 ![]({{.StaticRoot}}article_visual_debugger_in_vscode/red_breakpoint.png)
-debug for django will work due to the present .vscode/launch.json containing
+
```

Comment:
i have settings folder working as `.vscode` at Linux, does it work for you with `.VScode/launch.json` ?
I made a test, it fails for me with `.VScode` folder name to see debugging settings